src/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -251,6 +252,24 @@ public void run() {
         }
     }
 
+    /**
+     * Logs information about the scan.
+     * <p>
+     * It logs the {@link #nodeInScopeCount number of nodes} that will be scanned and the name of the {@link #user}, if any.
+     */
+    private void logScanInfo() {
+        StringBuilder strBuilder = new StringBuilder(150);

```

Comment:
Didn't add any check because that's the default level and it's not logged often. If you prefer I can add the check.